internal/core/src/mmap/Column.h

```diff
@@ -136,7 +167,7 @@ class ColumnBase {
 
         if (data_ != nullptr) {
             std::memcpy(data, data_, len_);
-            if (munmap(data_, cap_)) {
+            if (munmap(data_, cap_ + padding_)) {
                 AssertInfo(
```

Comment:
Is there some situation that unmap it, but others is still using it ?  Or multi-thread problem

Reply:
`Append()` is for load only, and the column is immutable after load